backend/database/migrations/2025_04_05_141946_create_cars_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('cars', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->string('description');
+            $table->string('registration_number');
+            $table->string('technical_details');
```

Comment:
w momencie jak nie podamy żadnych danych, to wrzucamy tu pustego stringa? Raczej poprawniej powinno być, żeby po prostu przekazywać nulla, więc tutaj nullable()